cart.spec

```diff
@@ -0,0 +1,82 @@
+Name:          cart
+Version:       0.0.1
```

Comment:
Duplicated with what's in Makefile.

Reply:
Yes.  There is some future work to have the `Makefile` get the version from the specfile.

Reply:
True.  One thing about scons, the way we define our SConstruct files they can be imported as python, or just run directly, so exporting the version from a version in the python code is actually really easy.